plugin-code/src/main/java/com/trivago/cluecumber/CluecumberReportPlugin.java

```diff
@@ -99,6 +102,11 @@ public void execute() throws CluecumberPluginException {
         // Initialize logger to be available outside the AbstractMojo class
         logger.setMojoLogger(getLog());
 
+        if (skip) {
```

Comment:
Exactly what I meant :-)

Reply:
> You can merge your changes and they will be included in the next release.

Thanks Benjamin   but I'm not authorized to merge pull requests :
![schermata 2018-10-04 alle 12 33 19](https://user-images.githubusercontent.com/5583341/46468805-d62a7880-c7d1-11e8-8dec-4f1a0be90473.png)